### ISSUE TITLE
Certain LDAP fields will not sync in, instead wiping out any stored values

### ISSUE THREAD

**Issue by aaronpollock** [grokability/snipe-it#11936]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

We have LDAP sync configured and a recurring sync that runs nightly to keep our users in Snipe-IT up to date. While the sync succeeds, certain fields are never imported and instead end up blanking out any manually entered values in the user's Snipe-IT profile:

- LDAP Employee Number
- LDAP Department
- LDAP Job Title

### Reproduction steps

1. Configure fields in LDAP sync with appropriate LDAP fields
2. Run a manual sync or Test LDAP sync on the config page
3. Employee Number in test is all "null", all listed fields are erased on full sync

### Expected behavior

We would expect the Title, Department, and Employee Number fields to be synchronized in from LDAP/AD.

### Screenshots

![image](https://user-images.githubusercontent.com/20303969/194369978-dae0b840-e4bb-4aa7-82e4-fb58cd01c3a1.png)

### Snipe-IT Version

v6.0.11 - build 8758

### Operating System

Ubuntu 18.04

### Web Server

Apache 2.4.29

### PHP Version

8.1.11

### Operating System

Windows

### Browser

Chrome

### Error messages

```shell
No error messages.
```

**Comment by uberbrady** [comment]
Those fields *should* work if you put the right values in. They need to be all-lowercase (due to a PHP implementation choice). What values are you using for those fields, and what directory are you using?

**Comment by aaronpollock** [comment]
We are using Active Directory for this, and I wrote a little basic php script to ensure everything is coming over - I can confirm all the fields and more come over when a successful ldap_get_attributes command is ran.

We are mapping:
employeeid -> LDAP Employee Number
department -> LDAP Department
title -> LDAP Job Title

All three fields exist for all existing users in AD.

**Comment by nastybutler** [comment]
For employee number make sure you put "employeenumber" into the LDAP Employee Number.  If you put "employeeNumber" like a normal LDAP search it will fail.  Took me a week to figure it out, and even then it was a lucky guess.

**Comment by aaronpollock** [comment]
Yes - we are doing "employeeid", all lower case even though the LDAP field is actually employeeId. No matter what we put there, it will return null.

**Comment by snipe** [comment]
Hi there - I know this is a pretty old issue, but I wanted to check and see if you ever got this resolved? If so, can you please close the ticket? If not, can you update us on what version of Snipe-IT you're currently using, so we can see if it's still an issue on latest?

:) Thanks!

**Comment by aaronpollock** [comment]
Yes, sorry, this was resolved. I honestly do not remember when, but we stay up to date - currenty running v8.2.1 build 19068 and it's working fine using Okta LDAP.

**Comment by snipe** [comment]
Thanks for letting us know!
